modules/tensor_mechanics/src/materials/ComputeFiniteStrainElasticStress.C

```diff
@@ -34,11 +37,28 @@ ComputeFiniteStrainElasticStress::initialSetup()
 }
 
 void
+ComputeFiniteStrainElasticStress::initQpStatefulProperties()
+{
+  ComputeStressBase::initQpStatefulProperties();
+
+  if ((isParamValid("initial_stress")) && !_elasticity_tensor_is_constant[_qp])
+    mooseError("A finite stress material cannot both have an initial stress and an elasticity "
+               "tensor with varying values; please use a defined constant elasticity tensor, "
+               "such as ComputeIsotropicElasticityTensor, if your model defines an initial stress");
+}
+
+void
 ComputeFiniteStrainElasticStress::computeQpStress()
 {
   // Calculate the stress in the intermediate configuration
-  RankTwoTensor intermediate_stress =
-      _stress_old[_qp] + _elasticity_tensor[_qp] * _strain_increment[_qp];
+  RankTwoTensor intermediate_stress;
+
+  // Check if the elasticity tensor has changed values
+  if (!_elasticity_tensor_is_constant[_qp])
```

Comment:
Oh, i think this is because of initial_stress - right?  Otherwise you'd get stress=0 at the start of the simulation, even if you used initial_stress = something.